### DISCUSSION TITLE
MeshCentral not working

### DISCUSSION THREAD

**Discussion by sda3-14** [amidaware/tacticalrmm#1912] [General]
We've just set up TacticalRMM, but we are unable to install any agents. It will always show a message saying `Unable to download the mesh agent from the RMM`.
It's also not possible to log in to MeshCentral, it always says the credentials are incorrect. I did already try disabling and reenabling `Sync Mesh Perms with TRMM`, but that did not make any difference.
Running `/rmm/api/env/bin/python /rmm/api/tacticalrmm/manage.py check_mesh`, it tells me that `you are using a custom mesh device group name. The name in TRMMs Global Settings > MeshCentral must match a MeshCentral group exactly.` - We have never changed this setting, however.
I have also noticed that while there is a reference to MongoDB on the Troubleshooting page (`sudo systemctl status mongod.service`), the service is completely nonexistant on my machine.
I have also already tried to do an `./update.sh --force`.
One thing that went wrong during installation is that I accidentally supplied an incorrectly formatted certificate, causing nginx to crash on startup. After replacing the certificate and restarting nginx, I was able to log in to the UI. I am unsure whether this might have had any repercussions for the install script.

**Comment by wh1te909** [comment]
try this https://github.com/amidaware/tacticalrmm/discussions/1114#discussioncomment-9729255

**Reply by sda3-14** [reply]
Unfortunately, that did not help. However, we tried creating a user in MeshCentral manually, and after logging in, there was in fact no device group. We tried creating a device group "TacticalRMM" and adding the automated user to it. After that, our users from TRMM were successfully synced into MeshCentral, and we were able to install agents. However, none of these agents are currently showing up in MeshCentral, and trying to remote into the the devices from TacticalRMM only shows a blank page...
